utils/ci/miri.sh

```diff
@@ -0,0 +1,19 @@
+set -ex
+
+if rustup component add miri ; then
```

Comment:
I'd prefer test failure if Miri fails to install.

Reply:
Are you sure? Not all nightlies come with Miri, so this would make CI fail whenever Miri's toolstate [goes to red](https://rust-lang.github.io/rustup-components-history/).

Reply:
I see what you mean. I guess we just have to check the runner occasionally to make sure it's doing something.